server/src/main/java/org/elasticsearch/index/query/SearchExecutionContext.java

```diff
@@ -110,7 +110,7 @@ public SearchExecutionContext(
         MapperService mapperService,
         MappingLookup mappingLookup,
         SimilarityService similarityService,
-        ScriptService scriptService,
+        ScriptCompiler scriptService,
```

Comment:
Note that these changes (replacing a reference to the concrete ScriptService class with one to its ScriptCompiler interface instead) make using scripts in tests a lot easier.